backend/Tournament/exceptions.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework.exceptions import APIException
+from rest_framework import status
+
+
+class TournamentSetupException(APIException):
```

Comment:
Just food for thought: 
How do we want to handle when someone starts creating a tournament, lets the session expire in the middle of it and without refreshing sends further requests (e.g. completing player data).
Are they just kicked out and have to start over?

Reply:
when they'd try to send a new request with the token that they'd get an error informing them that the token has expired.